peewee_extra_fields/__init__.py

```diff
@@ -162,8 +163,8 @@ def db_value(self, value):
                 self.algorithm, bytes(value, "utf-8"), self.salt,
                 self.iterations, self.dklen)).decode("utf-8")
 
-    @staticmethod
```

Comment:
This change removes a bug.
It is unrelated to the intent of this PR.
Needed to remove it to have a working version of this library.

Reply:
Awesome :grey_exclamation: